lib/Runtime/Language/ModuleNamespace.h

```diff
@@ -20,7 +20,11 @@ namespace Js
     public:
         static ModuleNamespace* GetModuleNamespace(ModuleRecordBase* moduleRecord);
 
-        static bool Is(Var ns) { return ModuleNamespace::Is(ns); }
+        static bool Is(Var ns)
+        {
+            // This function calls itself ?
+            return false; // ModuleNamespace::Is(ns);
```

Comment:
Should we change to

```c++
            AssertMsg(false, "ModuleNamespace::Is is not expected to be called");
            return false; // ModuleNamespace::Is(ns);
```

And check that in and check that we get no hits on it for a while before removing it altogether?